packages/@react-aria/selection/src/useSelectableCollection.ts

```diff
@@ -337,8 +337,8 @@ export function useSelectableCollection(options: SelectableCollectionOptions): S
       manager.setFocused(true);
       manager.setFocusedKey(focusedKey);
 
-      // If no default focus key is selected, focus the collection itself.
-      if (focusedKey == null && !shouldUseVirtualFocus) {
+      // If no default focus key is selected and the collection is mounted, focus the collection itself.
+      if (focusedKey == null && !shouldUseVirtualFocus && ref.current) {
```

Comment:
Hm, this makes sense for stopping the error but there might be an issue since then focus never gets set on the ListBox itself (this useEffect only happens on the initial render). That would mean the focus remains on the select button itself and pressing up/down would open and close the select menu instead of navigating through the items in the list.

Perhaps this should be extracted to it's own useEffect? Maybe useListBox shouldn't be pulled out into the Select component? Not sure

Reply:
So I had some similar reservations, however, in practice it seems to work, though the example code didn't have any of the focus/selection code running yet.
I updated the example to match our code on our documentation page so that I could try out the concern that focus would remain on the button, basically a copy paste from https://react-spectrum.adobe.com/react-aria/useSelect.html#example and now it would appear that this check is unnecessary. It behaves with or without it.

I think the outcome of this is
1) this person can update their code to be more like our documentation
2) we should still perform this check somewhere, but maybe some more thought should be put into it, i'll try some other configurations later hopefully this week